app/src/main/java/com/example/b07group7project/create_order/GetCartImplementation.java

```diff
@@ -0,0 +1,20 @@
+package com.example.b07group7project.create_order;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Example Class Only For Testing, Will be replaced with Database Call (or from cart)
+public class GetCartImplementation implements GetCartInterface{
+
+    @Override
+    public List<Product> getCart(String user) {
+        ArrayList<Product> products = new ArrayList<Product>();
+
+        for (int i = 0; i < 15; i++){
+            Product prod = new Product("FlashLight", 9.99, 5);

```

Comment:
For testing, include a few different product types so you can make sure your buttons call the correct item